subjects/java/projects/travel-plan/audit/README.md

```diff
@@ -0,0 +1,98 @@
+
+#### Comprehension
+
+##### Ask the student to describe the microservices architecture implemented.
+
+###### Have you clearly defined the boundaries for each microservice based on business domains?
+
+###### Do your microservices align directly with specific business functions?
+
+###### Are your microservices designed to operate independently of one another?
+
+###### Can each microservice be deployed, updated, and scaled without affecting others?
+
+###### Is your architecture designed to support the independent scalability of each microservice?
+
+###### Does your system maintain functionality even when one or more services fail?
+
+###### Is there an API Gateway in your architecture to manage incoming requests?
+
+###### Can you track and trace a request across multiple services easily?
+
+###### Are communications between your microservices secured?
```

Comment:
Nothing major, but I would add SSL/TLS protocol since they're mentioned in the subject.

Reply:
i don't see that i should mention that here since it's already mentioned at line 36

Reply:
Should this question be removed then, since it is asked at line 36 the same thing?